src/PKSim.Infrastructure/Reporting/Markdown/Builders/CompoundMarkdownBuilder.cs

```diff
@@ -14,13 +20,49 @@ public CompoundMarkdownBuilder(IMarkdownBuilderRepository markdownBuilderReposit
          _markdownBuilderRepository = markdownBuilderRepository;
       }
 
-      public override void Report(Compound compound, MarkdownTracker tracker)
+      public override void Report(Compound compound, MarkdownTracker tracker, int indentationLevel)
       {
-         tracker.Add(compound.Name.ToMarkdownTitle());
+         tracker.Add($"{PKSimConstants.ObjectTypes.Compound}: {compound.Name}".ToMarkdownLevelElement(indentationLevel));
+         var sublevelIndentation = indentationLevel + 1;
+         tracker.Add(PKSimConstants.ObjectTypes.Parameter.Pluralize().ToMarkdownLevelElement(sublevelIndentation));
 
-         var allDefaultAlternatives = compound.AllParameterAlternativeGroups().Select(x => x.DefaultAlternative).SelectMany(x => x.AllParameters()).Where(x => !x.IsDefault);
+         var allAlternatives = compound.AllParameterAlternativeGroups().SelectMany(x => x.AllAlternatives);
+         var allCompoundParameters = new List<CompoundParameter>();
+         foreach (var alternative in allAlternatives)
+         {
+            var allUserDefinedParameters = alternative.AllParameters(p => !p.IsDefault).ToList();
+            allUserDefinedParameters.Each(p => allCompoundParameters.Add(mapFrom(alternative, p)));
+         }
 
-         _markdownBuilderRepository.Report(allDefaultAlternatives, tracker);
+         var allSimpleParameters = compound.AllSimpleParameters().Where(p => !p.IsDefault).ToList();
+         allSimpleParameters.Each(p => allCompoundParameters.Add(mapFrom(p)));
+         tracker.Add(allCompoundParameters.ToMarkdownTable());
+
+         tracker.Add(PKSimConstants.UI.CalculationMethods.ToMarkdownLevelElement(sublevelIndentation));
+         _markdownBuilderRepository.Report(compound.CalculationMethodCache, tracker);
+
+         tracker.Add(PKSimConstants.UI.Processes.ToMarkdownLevelElement(sublevelIndentation));
+         compound.AllProcesses().Each(x => _markdownBuilderRepository.Report(x, tracker, sublevelIndentation + 1));
+      }
+
+      private CompoundParameter mapFrom(IParameter parameter) => parameter.To<CompoundParameter>();
+
+      private CompoundParameter mapFrom(ParameterAlternative alternative, IParameter parameter)
+      {
+         var compoundParameter = mapFrom(parameter);
+         compoundParameter.Alternative = alternative.Name;
+         compoundParameter.Alternative = alternative.Name;
+         compoundParameter.Default = alternative.IsDefault;
+         return compoundParameter;
+      }
+
+      private class CompoundParameter : IParameterElement
+      {
+         public string Name { get; set; }
```

Comment:
The table is created based on the property ordere.g. Name, Value, ValueOrigin etc..